packages/sd-jwt/src/repository/SdJwtRecord.ts

```diff
@@ -0,0 +1,98 @@
+import type { TagsBase, Constructable } from '@aries-framework/core'
+import type { DisclosureItem, HasherAndAlgorithm } from 'jwt-sd'
+
+import { JsonTransformer, Hasher, TypedArrayEncoder, BaseRecord, utils } from '@aries-framework/core'
+import { Disclosure, HasherAlgorithm, SdJwtVc } from 'jwt-sd'
+
+export type SdJwtRecordTags = TagsBase & {
+  disclosureKeys?: Array<string>
+}
+
+export type SdJwt<
+  Header extends Record<string, unknown> = Record<string, unknown>,
+  Payload extends Record<string, unknown> = Record<string, unknown>
+> = {
+  disclosures?: Array<DisclosureItem>
+  header: Header
+  payload: Payload
+  signature: Uint8Array
+
+  // TODO: include the holder key for key binding in here, for ease of use
```

Comment:
@TimoGlastra I would like to include the holder key that is bound to this SD-JWT. We can store the public key of the pair and derive it with Askar from there. Or we include the didurl and handle it like we do in the services now by resolving it and using askar to get the private key. 

Do you have a preference / other idea for this?

Reply:
Storing the `holderDidUrl` for now.